### PR TITLE
chore: remove redundant words in comment

### PR DESCRIPTION
**Pull request by lvyaoting**
### Description

remove redundant words in comment

### Applicable issues

- fixes #

### Checklist

- [ ] Test coverage for new or modified code paths
- [ ] Changelog is updated
- [ ] Required documentation changes (e.g., `docs/rpc/openapi.yaml` and `rpc-endpoints.md` for v2 endpoints, `event-dispatcher.md` for new events)
- [ ] New clarity functions have corresponding PR in `clarity-benchmarking` repo
- [ ] New integration test(s) added to `bitcoin-tests.yml`

### CODE REVIEW

File: clarity/src/vm/docs/mod.rs [hunk 1]
```diff
@@ -529,7 +529,7 @@ const LOG2_API: SimpleFunctionAPI = SimpleFunctionAPI {
     snippet: "log2 ${1:expr-1}",
     signature: "(log2 n)",
     description:
-        "Returns the power to which the number 2 must be raised to to obtain the value `n`, rounded 
+        "Returns the power to which the number 2 must be raised to obtain the value `n`, rounded
```
Comment (obycode) [inline clarity/src/vm/docs/mod.rs:532]: This one was correct as "to to". It is the two phrases "raised to" and "to obtain".

Comment (obycode) [reply clarity/src/vm/docs/mod.rs]: Ah, I'm sorry. I just re-read this and realized it was correct with just one "to".

**Comment by obycode** [comment]
Thanks for your contribution. Can you target this against `develop` instead of `master`?

**Comment by lvyaoting** [comment]
> Thanks for your contribution. Can you target this against `develop` instead of `master`?

Of course.

**Comment by lvyaoting** [comment]
> Thanks for your contribution. Can you target this against `develop` instead of `master`?

I closed this PR and resubmitted one for develop

**Comment by lvyaoting** [comment]
The new PR:  https://github.com/stacks-network/stacks-core/pull/5536

**Pull request closed without merging**